lib/core/src/Cardano/Wallet/Api/Types.hs

```diff
@@ -1223,17 +1219,32 @@ data ApiAnyCertificate n =
     deriving (Eq, Generic, Show)
     deriving anyclass NFData
 
-data ApiPolicyScript = ApiPolicyScript
+newtype ApiPostPolicyKeyData = ApiPostPolicyKeyData
+    { passphrase :: (ApiT (Passphrase "raw"))
+    }
+    deriving (Eq, Generic, Show)
+    deriving anyclass NFData
+
+data ApiTokenAsset = ApiTokenAsset
```

Comment:
Perhaps call this `ApiTokenAmountFingerprint`?

I feel the juxtaposition of `Token` and `Asset` might be confusing here.

(And it there's the fact that this also includes an amount and a fingerprint.)

Reply:
Fixed in 6951314468399a163fa58739c5b036bf1cbcb3b4.